addons/escoria-core/plugin.gd

```diff
@@ -2,6 +2,23 @@
 tool
 extends EditorPlugin
 
+# The path to node 'escoria'.
+const ESCORIA: String = "/root/escoria"
+
+
+# Returns the 'escoria' singleton/autoload.
+#
+# *Returns*
+# The escoria singleton.
+func _get_escoria():
+	var escoria_singleton = get_node(ESCORIA)
+	if escoria_singleton == null:
```

Comment:
```suggestion
	if not is_instance_valid(escoria_singleton):
```

Reply:
Also, I think it would make sense to make this into a true singleton instead of calling `get_node` with each call. I.e. make an instance variable (I realize we don't really have static variables in GDScript), check for null, populate it if it's null, and if it can't be loaded, throw an error. Then return the instance var.